ruby_on_rails/app_initialisation.md

```diff
@@ -1,61 +1,77 @@
 # Initialise the Rails App
 
-* Check if you are using the latest version of ruby supported by SemahoreCI and Heroku with `ruby -v` and update it if you are not.
-Run `rbenv versions` to check your installed ruby versions.
+## Default Rails setup
+
+* Check if you are using the latest version of ruby supported by SemahoreCI and Heroku with `ruby -v`
+and update it if you are not. Run `rbenv versions` to check your installed ruby versions.
 Follow the instructions on screen to eventually install it.
 
+* Switch the global Ruby version to the newest one with `rbenv global <ruby-version>` so that the latter commands
```

Comment:
I personally have a `.ruby-version` file in the Renuo projects folder

Reply:
I added the hint.